.env.example

```
@@ -51,16 +51,15 @@ GIT_SERVER_TOKEN="dev-git-token"
+# Either MASTER_KEY (inline base64) or MASTER_KEY_FILE (path to a file with base64) must be set.
+MASTER_KEY=""
+# MASTER_KEY_FILE="/var/lib/homelab-manager/master.key"
 
-# OpenBao (secrets management)
-# Use localhost for local dev (bun dev runs outside Docker).
-# Docker services override this to "http://openbao:8200" internally.
-OPENBAO_URL="http://localhost:8200"
-OPENBAO_TOKEN="dev-root-token"
+# Local-dev-only seeder: provisions a "localhost" managed host and dev keypair on first run.
+# Agent reads the public JWK from data/dev-agent-pubkey.json (volume-mounted by docker-compose.local.yml).
+HOMELAB_DEV_SEED="true"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)